tests/components/shelly/test_utils.py

```diff
@@ -224,3 +225,10 @@ async def test_get_rpc_input_triggers(mock_rpc_device, monkeypatch) -> None:
 
     monkeypatch.setattr(mock_rpc_device, "config", {"input:0": {"type": "switch"}})
     assert not get_rpc_input_triggers(mock_rpc_device)
+
+
+def test_get_release_url() -> None:
+    """Test get_release_url() with a device without a release note URL."""
+    result = get_release_url(1, "SHMOS-01", False)
+
+    assert result is None
```

Comment:
```suggestion
    assert result == GEN1_RELEASE_URL
```

Reply:
This test is for `SHMOS-01`, there is no change log for this device so the result should be `None`. But I agree this is misleading. I just added more parameters for this test.